docs/DeveloperGuide.md

```diff
@@ -182,10 +181,10 @@ the filtered transaction with a success message.
    Use Case ends
 
 **Extensions**
-- 2a. User enters an incorrect format.
-  - 2a1. UniCa$h displays an error message with the correct command format.
-  - Use case resumes at step 1.
 
+- 2a. User enters an incorrect format.
+    - 2a1. UniCa$h displays an error message with the correct command format.
+    - Use case resumes at step 1.
 
```

Comment:
Same here

Reply:
<img width="969" alt="image" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/dfcdfaa6-bc00-4995-a816-554eade60ea5">

The edited `- 2a1` is a few spaces away from the original. Probably IntellIj also. You can try to fix it if it's possible, if it's not I think it's fine. Just a minor concern about authorship, but a couple of lines shouldn't be an issue.